Mathlib/Data/Fintype/Sigma.lean

```diff
@@ -36,3 +36,15 @@ instance PSigma.fintype {α : Type*} {β : α → Type*} [Fintype α] [∀ a, Fi
     Fintype (Σ'a, β a) :=
   Fintype.ofEquiv _ (Equiv.psigmaEquivSigma _).symm
 #align psigma.fintype PSigma.fintype
+
+lemma Set.biUnion_finsetSigma_univ (s : Finset ι) (f : Sigma κ → Set α) :
+    ⋃ ij ∈ s.sigma fun _ ↦ Finset.univ, f ij = ⋃ i ∈ s, ⋃ j, f ⟨i, j⟩ := by aesop
+
+lemma Set.biUnion_finsetSigma_univ' (s : Finset ι) (f : ∀ i, κ i → Set α) :
```

Comment:
I am not a big fan of the notatiion `f : ∀ i, κ i → Set α`, but I guess `PiNotation` is not available here.

Reply:
It's available, actually, but I'm afraid using it is a lost cause. I can only find it in comments.